.editorconfig

```diff
@@ -13,13 +13,20 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
+indent_size = 4
+eclint_block_comment_start = /*
+eclint_block_comment = *
+eclint_block_comment_end = */
 
-[*.yml]
+[{*.yml,*.json}]
```

Comment:
The https://core.trac.wordpress.org/browser/trunk/.editorconfig file doesn't have this, as JSON files for WP-related projects should follow the same 4-space tabs for indentations. 

Can you amend this, and the indentation changes for `composer.json` please?

Reply:
Done.

Reply:
I see that you've added an entry for `[*.json]`, but this isn't needed - JSON files should inherit the indent_style=tab from `[*]`, and the default indent_size=4, just like the linked editorconfig file in WP core.

Reply:
Removed.